src/main/java/service/impl/FruitRecordParserServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package service.impl;
+
+import exception.InvalidInputException;
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitRecordDto;
+import service.FruitRecordParserService;
+import service.validator.DataValidator;
+
+public class FruitRecordParserServiceImpl implements FruitRecordParserService {
+    public static final String COMMA = ",";
+    public static final int PROBABLY_TITLE_INDEX = 0;
+    public static final int VALID_STRING_INDEX = 1;
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitRecordDto> getRecord(List<String> stringsFromFile) {
+        List<FruitRecordDto> outputList = new ArrayList<>();
+        DataValidator dataValidator = new DataValidator();
```

Comment:
make this variable belong to class instead of method, we don't need to create new instance per each method call